ci/python-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Upload Python Package
+
+on: [release]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      if: github.event.action == 'created'
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      if: github.event.action == 'created'
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      if: github.event.action == 'created'
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```

Comment:
Ideally this should use the new [token functionality](https://pypi.org/help/#apitoken)

Reply:
Though I agree using tokens is a better method, having the option to configure the username and password via secrets allows the user to choose between using their credentials or a token, depending on their use case.